src/Membership/MembershipRequest.php

```diff
@@ -170,7 +169,7 @@ public static function createFromAdherent(Adherent $adherent): self
         $dto->phone = $adherent->getPhone();
         $dto->emailAddress = $adherent->getEmailAddress();
         $dto->mandate = $adherent->getMandate();
-        $dto->elected = !\is_null($adherent->getMandate());
+        $dto->elected = \count($adherent->getMandate());
```

Comment:
What about adding `hasMandate()` method to return a boolean and use it here ?

Reply:
:heavy_check_mark:

Reply:
IMO it's better to have `hasMandate` because it's closer than `isElectedperson` of the attribute.